### PR TITLE
Don't pass form data to email

### PR DESCRIPTION
**Pull request by bendiggle**
In email-service.js constructor only set this.data if includeInEmail is true other set as empty string. This is so that emails can be sent with out the form data being in the email

### CODE REVIEW

File: .idea/compiler.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      <entry name="!?*.clj" />
+      <entry name="!?*.aj" />
+    </wildcardResourcePatterns>
+    <annotationProcessing>
+      <profile default="true" name="Default" enabled="false">
+        <processorPath useClasspath="true" />
+      </profile>
+    </annotationProcessing>
+  </component>
+</project>
```
Comment (joefitter) [inline .idea/compiler.xml]: @bendiggle 👎

**Review by joefitter — CHANGES_REQUESTED** [review]

**Review by JoeChapman — CHANGES_REQUESTED** [review]
what @joefitter said

**Comment by bendiggle** [comment]
@JoeChapman It was updated way before your comment!! @joefitter

**Review by JoeChapman — APPROVED** [review]

**Comment by JoeChapman** [comment]
@joefitter what say you?

**Pull request closed without merging**